actionpack/lib/action_controller/test_case.rb

```diff
@@ -754,8 +754,11 @@ def non_kwarg_request_warning
 
           Examples:
 
-          get :show, params: { id: 1 }, session: { user_id: 1 }
-          process :update, method: :post, params: { id: 1 }
+          BEFORE: get :show, id: 1, session: { user_id: 1 }
```

Comment:
More seriously, this would likely read significantly better as:

```
before:

get :show id: 1, session: { user_id: 1 }

after:

get :show, params: { id: 1 }, session: { user_id: 1 }
```